app/web_hub.go

```diff
@@ -203,7 +203,6 @@ func (a *App) PublishSkipClusterSend(message *model.WebSocketEvent) {
 
 func (a *App) InvalidateCacheForChannel(channel *model.Channel) {
 	a.Srv.Store.Channel().InvalidateChannel(channel.Id)
-	a.InvalidateCacheForChannelByNameSkipClusterSend(channel.TeamId, channel.Name)
 
 	if a.Cluster != nil {
```

Comment:
This whole block about sending the cluster invalidation message is no longer needed, this is done by the layer already.

Reply:
i deleted the whole if statement